### ISSUE TITLE
Fix FileSigned method in Filesystem module

### ISSUE THREAD

**Issue by ION28** [ION28/BLUESPAWN#201]
Currently having issues with some binaries in System32. Test by adding check for file signing to T1015

```cpp
// Check if Files are Signed
		for (auto key : vAccessibilityBinaries) {
			FileSystem::File file = FileSystem::File(L"C:\\Windows\\System32\\" + key);
			if (!file.GetFileSigned()) {
				reaction.FileIdentified(std::make_shared<FILE_DETECTION>(file.GetFilePath()));
				LOG_INFO(file.GetFilePath() << L" is not signed!");
				detections++;
			}
		}
```

**Comment by ION28** [comment]
potentially useful links: https://github.com/microsoft/Windows-classic-samples/tree/master/Samples/Win7Samples/security/cryptoapi/VerifyNameTrust

seems to be win8+: https://github.com/microsoft/Windows-classic-samples/blob/master/Samples/Security/CodeSigning/cpp/codesigning.cpp

**Comment by ION28** [comment]
Make sure to update T1015 when this is done

**Comment by ION28** [comment]
https://stackoverflow.com/a/45193834

**Comment by ION28** [comment]
https://www.pinvoke.net/default.aspx/wintrust.winverifytrust

**Comment by ION28** [comment]
closed in #280
